emmet-core/setup.py

```diff
@@ -40,10 +39,9 @@
             "matcalc>=0.0.4",
             "seekpath>=2.0.1",
             "robocrys>=0.2.8",
-            "pymatgen-analysis-defects>=2024.7.18",
-            "pymatgen-analysis-diffusion>=2024.7.15",
-            "pymatgen-analysis-alloys>=0.0.6",
-            "solvation-analysis>=0.4.1",
+            "pymatgen-analysis-diffusion>=2023.8.15",
```

Comment:
and do we need to downgrade the analysis libraries?

Reply:
This is a mistake introduced in the rebase, my apologies will fix